cockroachcloud/managing-cmek.md

```diff
@@ -72,16 +73,76 @@ Activate CMEK with a call to the clusters CMEK endpoint, using the cloud-specifi
 
 See the [API specification](../api/cloud/v1.html#operation/CockroachCloud_EnableCMEK).
 
-{% include_cached copy-clipboard.html %}
-```shell
-CLUSTER_ID= #{ your cluster ID }
-API_KEY= #{ your API key }
-curl --request POST \
-  --url https://cockroachlabs.cloud/api/v1/clusters/${CLUSTER_ID}/cmek \
-  --header "Authorization: Bearer ${API_KEY}" \
-  --header 'content-type: application/json' \
-  --data "@cmek_config.json"
-```
+1. Create a new file named `cmek_config.json`. This file will contain a JSON array of `region_spec` objects, each of which includes the name of a {{ site.data.products.db }} region and a `key_spec` that is specific to the target KMS platform and specifies the URI of the CMEK key and the credential that authorizes {{ site.data.products.db }} to encrypt and decrypt using the key.
+
+    Start from the example for your KMS platform and replace the placeholder values. Each of these examples includes `region_spec` objects for two {{ site.data.products.db }} regions; when enabling CMEK, you must include a `region_spec` for each region in the cluster.
+    - **AWS**:
+
+        {% include_cached copy-clipboard.html %}
+        ~~~ json
+        {
+          "region_specs": [
+            {
+              "region": "{COCKROACH_CLOUD_REGION}",
+              "key_spec": {
+                "type": "AWS_KMS",
+                "uri": "arn:aws:kms:{KEY_URI}",
```

Comment:
I would suggest to use an overall placeholder as {YOUR_AWS_KMS_KEY_ARN}. Specific value before a placeholder can make it confusing, and it's currently missing the region and AWS account id